psdk_wrapper/cfg/psdk_params.yaml

```diff
@@ -16,18 +16,18 @@
       camera_frame: "psdk_camera_link"
       publish_transforms: true
 
-      # Mandatory modules to be initialized. Mark with a true those which you 
-      # consider mandatory for your application, false otherwise. Be aware that 
-      # some modules might have inter-dependencies. Non mandatory modules will
-      # still be initialized but if case of failure, the node will continue to run.
+      # Mandatory modules to be initialized, choose among: telemetry,
+      # flight_control, camera, gimbal, liveview. Be aware that some modules might
+      # have inter-dependencies. Modules not listed here will be initialized but if they fail,
+      # the node will continue to run.
       mandatory_modules:
         telemetry: true
         flight_control: true
         camera: false
         gimbal: false
         liveview: false
-  
-      data_frequency: # Options are: 1, 5, 10, 50, 100, 200, 400 Hz 
+
```

Comment:
add HMS module with default value

Reply:
This does not apply here, as the DJI's HMS is publishing at a fixed frequency of **~1 Hz** (and thus, the callback function registered in this ROS2 node publishes at a similar frequency)

Reply:
not the data frequency, but the `mandatory_modules` array

Reply:
My bad, I didn't understand at first. Done!